packages/search-in-workspace/src/browser/search-in-workspace-result-tree-widget.tsx

```diff
@@ -985,10 +993,15 @@ export class SearchInWorkspaceResultTreeWidget extends TreeWidget {
             mode: 'reveal'
```

Comment:
If we want to mimic the behavior of the Navigator's opening, this should be `mode: preview ? 'reveal' : 'activate'`. Alternatively, `mode` can be omitted: the `EditorManager` will `activate`, the `EditorPreviewManager` will `reveal`.